Cable-Crimping/README.adoc

```diff
@@ -0,0 +1,43 @@
+= How to Crimp Rj45 cables
+
+== Strip the wire
+Make a shallow cut with a knife and run it around the cable 2.5 to 5 cm away from the end of the cable.  The jacket should slide off easily.
```

Comment:
Asciidoc best practices, one sentence one line. New lines don't do anything in asciidoc.